server/src/main/java/de/uftos/services/StudentService.java

```diff
@@ -114,17 +101,7 @@ public Student update(String id, StudentRequestDto studentRequest) {
     }
     Student student = studentRequest.map();
     student.setId(id);
-    this.repository.save(student);
-
-    List<StudentGroup> studentGroups =
```

Comment:
I guess this is thrown out as the relation between student groups and students is handled at the student group endpoint.

Reply:
yes